finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -156,6 +163,8 @@ private object ClassPath {
   }
 }
 
+object ignoredPaths extends GlobalFlag("", "Additional packages to be excluded from recursive directory scan")
```

Comment:
Instead of a String-flavored Flag, this should be a Seq[String] flavored flag.  That way you don't need to read the code to guess how to parse it.